python/pyarrow/__init__.py

```diff
@@ -90,7 +90,7 @@ def parse_git(root, **kwargs):
                          schema,
                          unify_schemas,
                          Array, Tensor,
-                         array, chunked_array, record_batch, table,
+                         array, chunked_array, record_batch, table, nulls,
```

Comment:
Can you add it to the API listing .rst page?

Reply:
Sure.